sqlalchemy_jsonapi/serializer.py

```diff
@@ -577,6 +577,7 @@ def delete_relationship(self, session, data, api_type, obj_id, rel_key):
                 if reverse_rel.direction == MANYTOONE:
                     permission = Permissions.EDIT
                 else:
+                    # pragma: no cover
```

Comment:
Yeah his tests never hit this piece of code either and I am not sure what to make of it.